config/bulle.yml

```diff
@@ -13,24 +13,35 @@ scrapers:
         value: "Ebullition"
       - name: "sourceUrl"
         value: "https://ebull.ch/fr"
+      - name: title
+        type: text
+        location:
+          - selector: div.eventListElement > div > p.artiste
+            all_nodes: true
+            separator: ", "
+      - name: url
+        type: url
+        location:
+          - attr: href
       - name: date
         type: date
+        on_subpage: url
         components:
           - covers:
               day: true
               month: true
-              time: true
```

Comment:
We'd need a time component here. If there's no time on the website anymore you can do something like this: https://github.com/jakopako/croncert-config/blob/main/config/lausanne.yml#L593

Reply:
found the time :)